.github/workflows/release-spec-prod.yml

```diff
@@ -0,0 +1,23 @@
+name: '(PROD) Release OpenAPI Spec'
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 9 * * *' # Run at 9AM UTC
+permissions:
+  contents: write
+jobs:
+  run-validations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      - name: Setup Node.js
+        uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b
+        with:
+          node-version: 20
```

Comment:
When doing global install this step is not required (nodejs is there).
This step gives nice cache capability so if you add spectral to be part of the nodejs package.json (which I strongly recomend) spectral bin will be callable in ./node_modules/bin folder.

Benefits: cuts job runtimes significantly and adds dependabot support.